vm/systemSmartContracts/validator.go

```diff
@@ -649,6 +649,10 @@ func (v *validatorSC) registerBLSKeys(
 	}
 
 	for _, blsKey := range newKeys {
+		if v.isNumberOfNodesTooHigh(registrationData) {
```

Comment:
if you have 60 nodes registered, but only 40 active, there will be no activateStakingFor for 10 nodes, even if you have stake for them. 
so it will behave different than having 40 nodes, and registering and staking another 20.

Reply:
that is true. but you can always unstake and unbond those nodes.

Reply:
ok